cirq-google/cirq_google/engine/abstract_processor.py

```diff
@@ -56,6 +56,9 @@ class AbstractProcessor(abc.ABC):
     async def run_async(
         self,
         program: cirq.Circuit,
+        *,
```

Comment:
nit: try to remove the `*` and see what breaks. if a few things break then we can fix them

Reply:
> nit: try to remove the `*` and see what breaks. if a few things break then we can fix them

This may be dangerous, because the main version had 2 optional string arguments at those positions.
Requiring everything to be a kwarg will make sure that string arguments are not mixed up.